### PR TITLE
Update text to clarify capabilities

### PR DESCRIPTION
**Pull request by V-Shannon**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/multi-geo-ediscovery-configuration.md [hunk 1]
```diff
@@ -17,16 +17,20 @@ description: Learn how to use the Region parameter to configure eDiscovery for u
 
 # Microsoft 365 Multi-Geo eDiscovery configuration
 
-[Advanced eDiscovery capabilities](https://docs.microsoft.com/microsoft-365/compliance/overview-ediscovery-20) allow a multi-geo eDiscovery administrator to search all of the geos without needing to utilize a "Region" security filter. Data is exported to the Azure instance of the central location of the multi-geo tenant. 
+With [advanced eDiscovery capabilities](https://docs.microsoft.com/microsoft-365/compliance/overview-ediscovery-20), a multi-geo eDiscovery manager or administrator can search all geos without needing to use a “Region” compliance security filter. In this case, data is exported to the Azure instance of the multi-geo tenant’s central location. Advanced eDiscovery is recommended for tenants whose central location is not in North America, Europe, or Asia Pacific and who need to perform eDiscovery across satellite geo locations.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-ediscovery-configuration.md]:
```suggestion
With [Advanced eDiscovery capabilities](https://docs.microsoft.com/microsoft-365/compliance/overview-ediscovery-20), a multi-geo eDiscovery manager or administrator can search all geos without needing to use a “Region” compliance security filter. In this case, data is exported to the Azure instance of the multi-geo tenant’s central location. Advanced eDiscovery is recommended for tenants whose central location is not in North America, Europe, or Asia Pacific and who need to perform eDiscovery across satellite geo locations.
```

File: microsoft-365/enterprise/multi-geo-ediscovery-configuration.md [hunk 2]
```diff
@@ -17,16 +17,20 @@ description: Learn how to use the Region parameter to configure eDiscovery for u
...
 
-Without advanced eDiscovery capabilities, an eDiscovery manager or administrator of a multi-geo tenant will be able to conduct eDiscovery only in the central location of that tenant. To support the ability to conduct eDiscovery for satellite locations, a new compliance security filter parameter named "Region" is available via PowerShell. This parameter can be used by tenants whose central location is in North America, Europe, or Asia Pacific. Advanced eDiscovery is recommended for tenants whose central location is not in North America, Europe, or Asia Pacific and who need to perform eDiscovery across satellite geo locations. 
+Without advanced eDiscovery capabilities, a multi-geo eDiscovery manager or administrator can only conduct eDiscovery in the tenant's central location. To conduct eDiscovery for satellite locations, the eDiscovery manager or administrator must use the “Region” compliance security filter. "Region" is available via PowerShell and can only be used by tenants whose central location is in North America, Europe, or Asia Pacific.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-ediscovery-configuration.md]:
```suggestion
Without Advanced eDiscovery capabilities, a multi-geo eDiscovery manager or administrator can only conduct eDiscovery in the tenant's central location. To conduct eDiscovery for satellite locations, the eDiscovery manager or administrator must use the “Region” compliance security filter. "Region" is available via PowerShell and can only be used by tenants whose central location is in North America, Europe, or Asia Pacific. 
```

File: microsoft-365/enterprise/multi-geo-ediscovery-configuration.md [hunk 3]
```diff
@@ -17,16 +17,20 @@ description: Learn how to use the Region parameter to configure eDiscovery for u
...
+Without advanced eDiscovery capabilities, a multi-geo eDiscovery manager or administrator can only conduct eDiscovery in the tenant's central location. To conduct eDiscovery for satellite locations, the eDiscovery manager or administrator must use the “Region” compliance security filter. "Region" is available via PowerShell and can only be used by tenants whose central location is in North America, Europe, or Asia Pacific. 
 
-The Microsoft 365 global administrator must assign eDiscovery Manager permissions to allow others to perform eDiscovery and assign a "Region" parameter in their applicable Compliance Security Filter to specify the region for conducting eDiscovery as satellite location, otherwise no eDiscovery will be carried out for the satellite location.
+To conduct eDiscovery for satellite locations using the “Region” compliance security filter, the Microsoft 365 global administrator must:
+- Assign the eDiscovery manager permission to allow others to conduct eDiscovery
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-ediscovery-configuration.md]:
```suggestion
- Assign the eDiscovery manager permission to allow others to conduct eDiscovery.
```

File: microsoft-365/enterprise/multi-geo-ediscovery-configuration.md [hunk 4]
```diff
@@ -17,16 +17,20 @@ description: Learn how to use the Region parameter to configure eDiscovery for u
...
+- Assign a "Region" parameter in the applicable Compliance Security Filter to specify which region is a satellite location for conducting eDiscovery
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-ediscovery-configuration.md]:
```suggestion
- Assign a "Region" parameter in the applicable compliance security filter to specify which region is a satellite location for conducting eDiscovery.
```

File: microsoft-365/enterprise/multi-geo-ediscovery-configuration.md [hunk 5]
```diff
@@ -17,16 +17,20 @@ description: Learn how to use the Region parameter to configure eDiscovery for u
...
+Otherwise, no eDiscovery will be conducted for the satellite location.
+
+Without advanced eDiscovery capabilities, when the eDiscovery manager or administrator role is set for a particular satellite location, the eDiscovery manager or administrator can only conduct eDiscovery for the SharePoint and OneDrive sites located in that satellite location. In this case, if an eDiscovery manager or administrator attempts to search SharePoint or OneDrive sites outside their specified satellite location, no results will be returned. Also, when the eDiscovery manager or administrator for a satellite location triggers an export, data is exported to that region's Azure instance. This helps organizations remain compliant by not allowing content to be exported across controlled borders.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-ediscovery-configuration.md]:
```suggestion
Without Advanced eDiscovery capabilities, when the eDiscovery manager or administrator role is set for a particular satellite location, the eDiscovery manager or administrator can only conduct eDiscovery for the SharePoint and OneDrive sites located in that satellite location. In this case, if an eDiscovery manager or administrator attempts to search SharePoint or OneDrive sites outside their specified satellite location, no results will be returned. Also, when the eDiscovery manager or administrator for a satellite location triggers an export, data is exported to that region's Azure instance. This helps organizations remain compliant by not allowing content to be exported across controlled borders.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@V-Shannon

**Comment by V-Shannon** [comment]
@yogkumgit The changes look all good on my end. What are the next steps to get this published?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT

**Comment by MikePlumleyMSFT** [comment]
@AdwoodMicrosoft can you review?

**Comment by yogkumgit** [comment]
@AdwoodMicrosoft Gentle reminder, kindly review as proposed by Mike

**Comment by MikePlumleyMSFT** [comment]
Closing per conversation with Shannon.

**Pull request closed without merging**